packages/grafana-prometheus/src/components/monaco-query-field/monaco-completion-provider/situation.ts

```diff
@@ -251,30 +252,8 @@ function getLabels(labelMatchersNode: SyntaxNode, text: string): Label[] {
     return [];
   }
 
-  let listNode: SyntaxNode | null = walk(labelMatchersNode, [['firstChild', LabelMatchList]]);
-
-  const labels: Label[] = [];
-
-  while (listNode !== null) {
-    const matcherNode = walk(listNode, [['lastChild', LabelMatcher]]);
-    if (matcherNode === null) {
-      // unexpected, we stop
-      return [];
-    }
-
-    const label = getLabel(matcherNode, text);
-    if (label !== null) {
-      labels.push(label);
-    }
-
-    // there might be more labels
-    listNode = walk(listNode, [['firstChild', LabelMatchList]]);
-  }
-
-  // our labels-list is last-first, so we reverse it
-  labels.reverse();
-
-  return labels;
```

Comment:
There is no LabelMatchList anymore https://github.com/prometheus/prometheus/pull/11333/files#top